### PR TITLE
toppingsData object updated

### PR DESCRIPTION
**Pull request by lukasznowak85**
please review or add your thoughts or merge :)

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -36,18 +36,118 @@
       }
     },
 
+    computed: {
+      toppings() {
+        return _.flattenDeep(_.map(this.toppingsData, function(category) {
```
Comment (kmswic) [inline src/App.vue]: This is a perfect place for an arrow function:
`_.map(this.toppingsData, category => category.toppings)`

File: src/components/Creator.vue [hunk 2]
```diff
@@ -10,12 +10,12 @@
   <div v-if="partipipcants.length">
     <h3>PiPca nie może zawierać:</h3>
     <p>
-      <!--<span v-for="topping in prohibited">{{ topping }}{{ $index }} </span>-->
-      {{ prohibited.join(', ') }}
+      <span v-for="topping in prohibited">{{ topping.name }} </span>
```
Comment (kmswic) [inline src/components/Creator.vue]: This is OK for me, but if we wanted to still have the commas, we could probably do something like e.g.
```js
{{ prohibited.map(t => t.name).join(', ') }}
```
as we can have any single expression inside `{{ }}`.

Comment (lukasznowak85) [inline src/components/Creator.vue]: Unfortunately it doesn't work. I get '[Vue warn]: Invalid expression. Generated function body:  scope.prohibited.map(scope.t=>scope.t.name).join(', ')' in console :( (?)

File: src/App.vue (hunk 1 above)
Comment (lukasznowak85) [inline src/App.vue]: Good point, changed.

**Pull request merged**